designcompose/src/main/java/com/android/designcompose/Shape.kt

```diff
@@ -0,0 +1,620 @@
+package com.android.designcompose
```

Comment:
Need to add that copyright blurb here

Reply:
Just run ./dev-scripts/format-all.sh

Reply:
Ok, I've added the copyright blurb to the new file using the `./dev-scripts/format-all.sh` tool :).